apps/web/lib/features/team/user-team-card/task-skeleton.tsx

```diff
@@ -61,15 +61,15 @@ export function UserTeamCardHeader() {
 					<div className="w-4 self-stretch border-l-[0.125rem] border-l-transparent " />
 					<div className="2xl:w-48 3xl:w-[12rem] w-1/5 lg:px-4 !pl-6 lg:!pl-8  flex flex-col items-center text-center  justify-center">
 						<Tooltip label={t('task.taskTableHead.WORKED_ON_TASK_HEADER_TOOLTIP')}>
-							{t('task.taskTableHead.TASK_WORK.TITLE')} <br /> {t('common.TASK')}
+							{t('common.TASK')} Time
 						</Tooltip>
 					</div>
 					<div className="w-1 self-stretch border-l-[0.125rem] border-l-transparent" />
 					<div className="w-1/5 lg:px-3 2xl:w-52 3xl:w-64 !pl-14 text-center">{t('common.ESTIMATE')}</div>
 					<div className="w-1 self-stretch border-l-[0.125rem] border-l-transparent" />
 					<div className="flex justify-center items-center cursor-pointer w-1/5 gap-4 lg:px-3 2xl:w-52 max-w-[13rem] !pl-14 text-center">
 						<Tooltip label={t('task.taskTableHead.TOTAL_WORKED_TODAY_HEADER_TOOLTIP')}>
-							{t('task.taskTableHead.TOTAL_WORK.TITLE')} <br /> {t('common.TODAY')}
+							Total {t('common.TODAY')}
```

Comment:
@CREDO23 hm, why we are doing this change? Now "Total" become hardcoded and will NOT be translated!